person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(name, age = 'unknown', parent_permision: true)
+    @name = name
+    @age = age
+    @parent = parent_permision
+  end
+
+  attr_accessor :@name, :@age
+
+  att_reader :@id
+
+  def can_use_services?
+    of_age || parent_permision
+  end
+
+  private
+
+  def of_age?
+    of_age >= 18
+  end
```

Comment:
- [ ]  kindly update your can_use_services method to use the instance variable @parent 

```Ruby
  def can_use_services?
    of_age || @parent
  end
```